tests/test_cases/test_iteration_mixedlang/test_iteration.py

```diff
@@ -60,6 +60,22 @@ def test_drivers(dut):
         raise TestFailure("No drivers found for dut.i_verilog.uart1.uart_rx_1.rx_data")
 
 
+@cocotb.test(expect_fail=True)
+def test_loads(dut):
+    """
+    Try iterating over loads of a signal.
+
+    Seems that few simulators implement vpiLoad
+    """
+    tlog = logging.getLogger("cocotb.test")
+    yield Timer(100)
+    for load in dut.i_verilog.uart1.ser_in.loads():
+        tlog.info("Found %s" % repr(load))
+        break
+    else:
+        raise TestFailure("No loads found for dut.i_verilog.uart1.ser_in")
```

Comment:
Is this intended? Print the first result and exit? Even if it is, it looks a bit odd.

I might suggest.
```suggestion
    loads =  dut.i_verilog.uart1.ser_in.loads()
    assert loads, "No loads found for dut.i_verilog.uart1.ser_in"
    for load in loads:
        tlog.info("Found %s" % repr(load))
```

Reply:
It's indeed a strange test, maybe buggy in the path that has never been taken since no simulator supports it.